### PR TITLE
3128 remove cachebreaker

### PR DESCRIPTION
**Pull request by IgorNadj**

### CODE REVIEW

File: packages/devops/configs/servers.conf [hunk 1]
```diff
@@ -462,6 +462,12 @@ server {
   include /etc/nginx/h5bp/basic.conf;
 
   location / {
+
```
Comment (IgorNadj) [inline packages/devops/configs/servers.conf:465]: Copied from https://stackoverflow.com/a/2068407

**Review by edmofro — APPROVED** [review]
lgtm :-)

**Comment by IgorNadj** [comment]
@edmofro added CORS header, was going to put it in a separate PR but easier to put it here

**Review by edmofro — APPROVED** [review]
Still lgtm

**Pull request merged**